docker/nginx/loris.nginx.conf

```diff
@@ -6,8 +6,8 @@ http {
   server {
     listen 9000;
 
-    location ~ ^/image/V[0-9]{4}.* {
-      rewrite ^/image/(V([0-9]{4}).*) /V$2000/$1 break;
+    location ~ "^/image/V[0-9]{4}.*" {
+      rewrite "^/image/(?<filename>(?<shard>V[0-9]{4}).*)" "/s3:${shard}000/${filename}" break;
```

Comment:
Can we add a quick comment explaining what sort of URLs we're trying to match, and how we're rewriting them? Make the regex comprehensible to the next person who comes along. :)